pkg/dm/spec/paths.go

```diff
@@ -14,24 +14,8 @@
 package spec
 
 import (
-	"fmt"
+	"github.com/pingcap/tiup/pkg/meta"
```

Comment:
Prefer to delete this file.

Reply:
delete in https://github.com/pingcap/tiup/pull/554/commits/85e238f7b3cfc2daa91158971fa89eea9d3ed108